storage/store.go

```diff
@@ -1267,16 +1264,14 @@ func raftEntryFormatter(data []byte) string {
 }
 
 // updateStoreStatus updates the store's status proto.
-func (s *Store) updateStoreStatus() error {
+func (s *Store) updateStoreStatus() {
 	now := s.clock.Now().WallTime
 	s.status.UpdatedAt = now
-	s.status.RaftIDs = []int64{}
-	for raftID := range s.ranges {
-		s.status.RaftIDs = append(s.status.RaftIDs, raftID)
-	}
+	s.status.RangeCount = int32(len(s.ranges))
 	key := engine.StoreStatusKey(int32(s.Ident.StoreID))
-	if err := engine.MVCCPutProto(s.engine, nil, key, proto.ZeroTimestamp, nil, s.status); err != nil {
-		return err
+	s.status.Stats = proto.MVCCStats(s.scanner.Stats().MVCC)
+	// TODO(bram): Update this so that it only puts with a Zero Timestamp.
```

Comment:
Oh right....hmm, there's currently no way to write non-MVCC values through the KV interface.

That's just as well. Remove this TODO. Let's actually keep historical values here. The number of stores in clusters is relatively small and we'll only be keeping around 144 values per store (every 10m of each day).